Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,79 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array
+     */
+    public function getMethodAttributes($index)
+    {
+        $tokens = $this->tokens;
+        $token = $tokens[$index];
+
+        if (!$token->isGivenKind(T_FUNCTION)) {
```

Comment:
> We could also make this into a more generic function

Exactly, if we need it then we should made it generic.
For now, we don't have `isMethod` vs `isFunction`. So even if we leave it as is, then we still could test `isLambda`